pom.xml

```diff
@@ -599,6 +599,11 @@
               <!-- Due to the removal of deprecated methods -->
               <exclude>org.apache.parquet.arrow.schema.SchemaMapping</exclude>
 
+              <!-- Make static variables final -->
+              <exclude>org.apache.parquet.avro.AvroReadSupport#AVRO_REQUESTED_PROJECTION</exclude>
```

Comment:
These are indeed breaking changes from the perspective of API but I think it makes sense to add final keyword to them.